package.json

```diff
@@ -13,7 +13,8 @@
   "dependencies": {
     "@nuxt/content": "^1.14.0",
     "core-js": "^3.15.1",
-    "nuxt": "^2.15.7"
+    "nuxt": "^2.15.7",
+    "vueperslides": "^3.3.1"
```

Comment:
I see how this ended up here, but actually it's not really part of this PR. I'm okay with leaving it in for now.

Reply:
I think this is when I tested the carousel, the package is added automatically. Let's leave it for now.